rfcs/0001-notary-integration.md

```diff
@@ -0,0 +1,50 @@
+# Problem
+kpack does not handle image signing.
+
+# Outcome
+kpack integrates with notary to sign images that it builds.
+
+# Actions to take
+There are a few concrete steps to make this work in kpack:
+1. Add a new section to the image config that allows users to configure notary settings:
+  ```
+  notary:
+    host: <notary server url>
+    rootPrivateKeyRef: <secret-name>
+    rootPublicKeyRef: <secret-name> (may not be necessary)
+    targetPrivateKeyRef: <secret-name>
+    targetPublicKeyRef: <secret-name> (may not be necessary)
+  ```
+2. Add a new container after export that uses the notary go client to sign the image. The presence or absence of the notary section in the image config will determine if the notary signing container is executed.
+
+There are a few different options to consider for the root and target keys:
+
+**Key generation (2 Options)**
```

Comment:
How will we pick an option? Will we decide before the rfc gets merged or when implementing?